exp/performance.md

```diff
@@ -0,0 +1,52 @@
+
+In addition, check your applications and make sure they use the resources in a fair way. Applications should avoid spikes in GPU utilisation, because such spikes require the application to reserve more resources and therefore reduce the container density.
+
+Generally, applications should use the smallest suitable instance type. However, if you see performance degradation when running the application, using a more powerful instance type usually helps (even though it reduces the container density). As an example, consider an application that runs on an Anbox Cloud deployment that does not have any GPUs installed. In this case, the rendering workload is put on the CPU instead of the GPU, and if the instance type of the application does not have a sufficient number of vCPU cores, the performance of the application is impacted. This can show, for example, in the virtual keyboard being really slow. By switching to a more powerful instance type, the container density is reduced, but the performance of each application container is increased.
+
+<a name="hardware-setup"></a>
+## Hardware and network setup
+
+See [Requirements](https://discourse.ubuntu.com/t/installation-requirements/17734) for the minimum hardware requirements for Anbox Cloud. Note that these list the minimum requirements, and using more powerful hardware will increase performance.
+
+The overall performance depends not only on the hardware used for the actual Anbox Cloud deployment, but also on the setup used for other components that Anbox Cloud relies on. For example, the etcd database must use a hard disk that is fast enough; see [Hardware recommendations](https://etcd.io/docs/v3.5/op-guide/hardware/) for detailed information.
+
+Also make sure that there is a stable network connection between the nodes of your cluster, to decrease the latency between nodes.
+
+<a name="startup-time"></a>
+## Container startup time
+
+A very noticeable performance issue is a long wait time when starting an application.
+
+When a user starts an application, Anbox Cloud retrieves the application image and launches a new container for it. By default, Anbox Cloud turns off image compression in LXD when launching a container from an image. This method speeds up the launch of the container (because the image does not need to be uncompressed), but it causes more traffic over the network (because the image is transferred uncompressed). If the network connection between your cluster nodes is rather slow, the overall container startup time might improve by enabling image compression. You can change the default configuration by setting the [images_compression_algorithm](https://charmhub.io/ams-lxd/configure#images_compression_algorithm) configuration on the `ams-lxd` charm.
```

Comment:
`...  the overall container startup time might improve by enabling image compression.`
->
`... the overall container startup time might improve by enabling image compression. Having a smaller image size also helps to synchronize the image across the LXD nodes in a cluster`
I feel it's better if we could mention the benefit of images with a smaller size being used in a cluster for synchronization.

Reply:
Your main point here is the smaller image size, not the image being compressed (even though this will of course also reduce the size) - correct?

Reply:
Compressed images result in a smaller image size, which would benefit the image synchronization across the cluster if the network condition is not good.